requirements.txt

```diff
@@ -5,3 +5,5 @@ requests==2.20.0
 retry==0.9.2
 tqdm==4.46.0
 waiting==1.4.1
+libvirt-python==4.5.0
```

Comment:
According to the pre-commit rules - this should have been ordered alphabetic. :)
Because your removed it from skipper - it didn't run on your commit.
I suggest for you to install pre-commit python package on your computer as well